diagnostics-extension/src/common

```diff
@@ -0,0 +1 @@
+../../common
```

Comment:
How about move `common` into `diagnostic-app` in this CL and link this to `../../diagnostic-app/common`, so that you can change less code when removing extension in the future.

Reply:
Done. PTAL!